iogt_users/tests.py

```diff
@@ -3,15 +3,19 @@
 from wagtail.core.models import Site
 
 from home.factories import SiteSettingsFactory, HomePageFactory
-from iogt_users.factories import UserFactory
+from iogt_users.factories import UserFactory, AdminUserFactory, GroupFactory
 from wagtail_factories import SiteFactory
 from questionnaires.factories import SurveyFactory
+from django.contrib.auth.models import Permission
 
 
 class PostRegistrationRedirectTests(TestCase):
     def setUp(self):
         self.user = UserFactory(has_filled_registration_survey=False)
-        self.admin_user = UserFactory()
+        self.admin_user = AdminUserFactory(has_filled_registration_survey=False)
+        self.admin_access_permission = Permission.objects.get(codename='access_admin')
+        self.group = GroupFactory()
+        self.group.permissions.add(self.admin_access_permission)
```

Comment:
I don't like that there are two different users being created in this common setup method but they are not both used in every test. I think this could be organised better:

- Set up the user in each test?
- Create a base class for common setup and a test class for regular users and another for admin users?

Only some variables need to be assigned to self e.g. `self.group` is only used in one test, `self.admin_access_permission` is not used outside of the setup method